libs/application/templates/secondary-school/src/fields/overview/CustodianOverview.tsx

```
@@ -91,60 +102,29 @@
-              <GridColumn span="1/2"></GridColumn>
-            )}
           </GridRow>
         )}
 
-        {(showMainOtherContact || showOtherContacts) && (
+        {!!contacts.length && (
           <GridRow marginTop={3}>
-            {showMainOtherContact && (
-              <GridColumn span={showOtherContacts ? '1/2' : '1/1'}>
+            {contacts.map((contact, index) => (
+              <GridColumn span={contacts.length > 1 ? '1/2' : '1/1'}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)